packages/synapse-react-client/src/components/dataaccess/AccessRequirementTable.integration.test.tsx

```diff
@@ -313,13 +315,15 @@ describe('Access Requirement Table tests', () => {
       ),
     )
 
-    // clicking a different column (name) should sort by that column, descending
-    const nameSortButton = screen.getByLabelText('Sort by Name')
+    // clicking a different column (name) should sort by that column, ascending
+    const nameSortButton = screen.getByLabelText(
+      'Sort by Access Requirement Name',
+    )
     await userEvent.click(nameSortButton)
     await waitFor(() =>
       expect(onServiceRecievedRequest).toHaveBeenLastCalledWith(
         expect.objectContaining({
-          sort: [{ field: 'NAME', direction: 'DESC' }],
+          sort: [{ field: 'NAME', direction: 'ASC' }],
```

Comment:
@tanstack/react-table is kinda smart, it makes the default initial sort for strings ascending, which is more intuitive for the user. Update the test accordingly.